public class Excepcion2 {

    public static void main(String[] args) {
        System.out.println("***Ejemplo de excepción 2***");
        int a=5;
        int b=0;
        int c=0;
        try {
            c=a/b;
            System.out.println("c="+c);
        }
        catch(ArithmeticException ex) {
        System.out.println("Atrapó a ArithmeticException: "+ex.getMessage());
        ex.printStackTrace();    
        }
        
    }
    
}
